p2p/src/authenticated/discovery/actors/spawner/actor.rs

```diff
@@ -132,9 +132,18 @@ impl<E: Spawner + Clock + Rng + CryptoRng + Metrics, O: Sink, I: Stream, C: Publ
                             let is_dialer = matches!(reservation.metadata(), Metadata::Dialer(..));
                             let info_verifier = self.info_verifier.clone();
                             move |context| async move {
+                                // Get greeting from tracker (returns None if not eligible)
```

Comment:
We don't resign info, so we should be able to cache this and avoid frequent lookups (although, we do need to know if the peer should be shutdown).

Reply:
I actually prefer leaving this as-is in case we start refreshing this.